### PR TITLE
pozdrawiam serdecznie kc <3

### PR DESCRIPTION
**Pull request by tapczanek1321**
admin pls accept??

### CODE REVIEW

**Review by goteusz-maszyk — APPROVED** [review]

**Comment by Sanielia** [comment]
Mild cringe, but okey

File: wazneinfo.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Teraz Gimpson o CS'ie, czyli wszystko gra
+Wpierdalam się do mainstreamu jak na Bombsite A
+I tak tak tak weź sprawdź mnie
+Bo pozamiatam łaków niczym piach na de_dustcie
+Ha! To jest hardkorowa muza
+Bomba której nie rozbroisz choćbyś i użył defuse'a
+Gimpson Mamiko tak jest kurwa
```
Comment (Norbiros) [inline wazneinfo.md]: Bardzo brzydkie słownictwo

**Review by Sanielia — APPROVED** [review]

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: wazneinfo.md (hunk 1 above)
Comment (tapczanek1321) [reply wazneinfo.md]: no i co mam z tym zrobic

Comment (Sanielia) [reply wazneinfo.md]: Po prostu dodaj cenzure

Comment (Norbiros) [reply wazneinfo.md]: Motyla noga

**Review by Sanielia — CHANGES_REQUESTED** [review]
Add extra enters, do it as a quote or something, because it doesn't display well, I mean, everything is in one line

**Review by Sanielia — APPROVED** [review]

**Review by Norbiros — APPROVED** [review]

**Pull request merged**